cmake/bgfxToolUtils.cmake

```
@@ -576,7 +576,12 @@ if(TARGET bgfx::shaderc)
 		cmake_parse_arguments(ARGS "${options}" "${oneValueArgs}" "${multiValueArgs}" "${ARGN}")
 
 		set(PROFILES 120 300_es spirv)
-		if(UNIX AND NOT APPLE)
+		if (IOS)
```

This is probably what's failing cmake-format
```suggestion
		if(IOS)
```